public/locales/es/translation.json

```diff
@@ -0,0 +1,821 @@
+  "We're happy you're here! Please take a moment to explore this page to see what's alive in our group. Introduce yourself by clicking Create to post a Discussion sharing who you are and what brings you to our group. And don't forget to fill out your profile - so likeminded new friends can connect with you!": "",
+  "We're working on expanding\n#topics to more places": "",
+  "We've sent a 6 digit code to {{email}}. The code will expire shortly, so please enter it here soon.": "",
+  "Website": "",
+  "Website for {{group.name}}": "",
+  "Weekly": "",
+  "Welcome": "",
+  "Welcome Message": "",
+  "Welcome to Hylo": "",
+  "Welcome to Hylo!": "",
+  "Welcome to {{group.name}}!": "",
+  "What I'm Learning": "",
+  "What I'm learning": "",
+  "What are you doing together?": "",
+  "What area does your group cover?": "",
+  "What do you need? What are you offering?": "",
+  "What do you want to see on the map?": "",
+  "What post types to display?": "",
+  "What questions are asked when a group requests to join this group?": "",
+  "What skills and interests are most relevant to your group?": "",
+  "What skills and interests are particularly relevant to this group?": "",
+  "What skills and interests do you have?": "",
+  "What skills do you want to learn?": "",
+  "What was wrong?": "",
+  "What would you like to create?": "",
+  "What you will do at your next event?": "",
+  "When people on Hylo need help or want to report a bug, they are interacting with a service called intercom. Intercom stores cookies in your browser to keep track of conversations with us, the development team.": "",
+  "When you select a prerequisite group, people must join the selected groups before joining <strong>{{name}}</strong>. Only parent groups can be added as prerequisite groups.": "",
+  "Where do you call home?": "",
+  "Where is your {{type}} located?": "",
+  "Which of the following skills & interests are relevant to you?": "",
+  "Who is able to see <strong>{{name}}</strong>?": "",
+  "Why was this {{type}} '{{selectedCategory}}'{{required}}?": "",
+  "Will open this URL in a new tab": "",
+  "With:": "",
+  "Word used to describe a group Moderator": "",
+  "Write something...": "",
+  "You": "",
+  "You already have a stripe account linked to this account. If you would like to link a different account, click the button below.": "",
+  "You are a member of": "",
+  "You are here!": "",
+  "You are requesting that <strong>{{group.name}}</strong> become a member of <strong>{{parentGroup.name}}</strong>": "",
+  "You can only select up to {{MAX_TOPICS}} topics": "",
+  "You can select a CSV file to import posts into {{name}}. Posts will be created by you. The file must have columns with the following headers:": "",
+  "You don't have any messages yet": "",
+  "You have no active messages": "",
+  "You have unsaved changes, are you sure you want to leave?": "",
+  "You have unsaved changes. Are you sure you want to leave?": "",
```

Comment:
we dont need these two version of the same things, lets use the one with the comma